codec/x264/encoder.go

```diff
@@ -56,7 +56,7 @@ func (v *encoder) Read() (img image.Image, release func(), err error) {
 }
 
 // Encode asks the codec to process the given image.
-func (v *encoder) Encode(_ context.Context, img image.Image) ([]byte, error) {
+func (v *encoder) Encode(ctx context.Context, img image.Image) ([]byte, error) {
```

Comment:
Why this change? It still looks like `ctx` isn't used anywhere.

Reply:
Good catch, I forgot to revert that change